pixel-point/pixelpoint-website
# feat: replace react-helmet with Gatsby-Head

This pull request replaces react-helmet with [Gatsby Head API](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-head/)

Additional updates:
- upgrade dependencies
- uninstall gatsby-plugin-react-helmet

**Checklist**:

<details>
<summary>Pages checked in Chrome Before / After(without hydration) / After with hydration:</summary>

- [x] index;
- [x] 404;
- [x] web-design;
- [x] web-development;
- [x] blog;
- [x] blog-post;
- [x] case-studies;
- [x] case-study;

**Screenshots**
Before:
![before](https://user-images.githubusercontent.com/75138180/195840850-40afef6f-45ad-436c-92db-7aa38e30e6e5.png)

After (without hydration):
![after (without hidration)](https://user-images.githubusercontent.com/75138180/195840887-649dc60c-6c79-44ef-a7de-6f35e3d1271a.png)

After:
![after (with hidration)](https://user-images.githubusercontent.com/75138180/195840904-5f4f8595-7746-4d42-9a31-6971c53e0ce9.png)

</details>

**Steps to test**
1. Open the [page](https://pixelpointwebsite-gatsbyhead.gtsb.io/)
2. Confirm that everything looks and works as expected
